redox/response_new_notes.go

```diff
@@ -137,59 +162,61 @@ func (n *NewNotes) SetProcedureFromOrder(order models.NewOrder) {
 					Room *string `json:"Room"`
 					Type *string `json:"Type"`
 				} `json:"Location,omitempty"`
-				PhoneNumber *struct {
-					Office *string `json:"Office"`
-				} `json:"PhoneNumber,omitempty"`
-				Role *struct {
-					Code        *string `json:"Code"`
-					Codeset     *string `json:"Codeset"`
-					Description *string `json:"Description"`
-				} `json:"Role,omitempty"`
-			} `json:"AdditionalStaff,omitempty"`
-			Location *struct {
-				Bed                   *string `json:"Bed"`
-				Department            *string `json:"Department"`
-				DepartmentIdentifiers *[]struct {
-					ID     *string `json:"ID"`
-					IDType *string `json:"IDType"`
-				} `json:"DepartmentIdentifiers,omitempty"`
-				Facility            *string `json:"Facility"`
-				FacilityIdentifiers *[]struct {
-					ID     *string `json:"ID"`
-					IDType *string `json:"IDType"`
-				} `json:"FacilityIdentifiers,omitempty"`
-				Room *string `json:"Room"`
-				Type *string `json:"Type"`
-			} `json:"Location,omitempty"`
-			PatientClass  *string `json:"PatientClass"`
-			VisitDateTime *string `json:"VisitDateTime"`
-			VisitNumber   *string `json:"VisitNumber"`
-		}{
-			AdditionalStaff: &additionalStaff,
+				PatientClass  *string `json:"PatientClass"`
+				VisitDateTime *string `json:"VisitDateTime"`
+				VisitNumber   *string `json:"VisitNumber"`
+			}{}
+			n.Visit = &visit
 		}
+		n.Visit.AdditionalStaff = &additionalStaff
```

Comment:
Should this be an append?

In the event that there's an existing `AdditionalStaff`, it would be overwritten. Is that the desire?